app/src/main/res/values/strings.xml

```diff
@@ -534,14 +534,14 @@
 
   <string-array name="faq_question_answers">
         <item>What image file formats are supported?#####The app supports jpeg, jpg, tiff, gif, psd, bmp, eps, png, etc.</item>
-        <item>Can I create a new text file using the app?#####No, you cannot. You can only convert already created text file to PDF.</item>
+        <item>Can I create a new text file using the app?#####No, you cannot. You can only convert an already created text file to PDF.</item>
         <item>Is there a limit on number of images I can add at once to create a PDF?#####No, there are no restrictions.</item>
         <item>Is there a limit to the image size I can add?#####No, there are no limits. However, we advise to restrict the size to less than 10MB. The conversion of multiple large image files take upwards of 10 seconds.</item>
         <item>Can I insert captions in the image after adding?#####No, you cannot.</item>
-        <item>Can I read PDF files using the app?#####No, you cannot as of now. This app doesn't work as a PDF viewer.</item>
-        <item>I want to access the PDF files I created externally outside the software, where are the pdf files saved?#####The pdf files created with the software are automatically saved to a location on your device storage name "PDFfiles"</item>
-        <item>How to change the default storgae location?#####To change the default storage location, \n1) Click the Hamburger icon (three horizontal line) at the top-left corner. \n2) Scroll down and click Settings. \n3) In the settings page, click the field just beneath the text 'Change storage location'. \n4) Open the preferred save location and click the Select icon at the top. \nOnce changed, the files created using the app will be stored in the new location.</item>
-        <item>How can I share the PDF files that are edited/created using the app?#####You can access and share the PDF files which you've edited/created directly from the app. To share the PDF files, \n1) Click the View Files option on the home page to view all the PDF files you have created. \n2) Select the file(s) you want to share using the checkbox. \n3) Next, Click the More option (three vertical dots) at the top-right and click Share. \n4) Select the desired method to share the files.</item>
+        <item>Can I read PDF files using the app?#####No, you cannot as of now. This app does not work as a PDF viewer.</item>
+        <item>I want to access the PDF files I created externally outside the software, where are the pdf files saved?#####The PDFfiles files created with the app are automatically saved in the folder named PDFfiles on your device storage.</item>
```

Comment:
The files are stored in the location provided by you in the settings section. By default, they are stored in the folder named PDFfiles on your device storage